src/text-filter/analytics-metadata/interfaces.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { GeneratedAnalyticsMetadataInputClearInput } from '../../input/analytics-metadata/interfaces';
+export type GeneratedAnalyticsMetadataTextFilterClearInput = GeneratedAnalyticsMetadataInputClearInput;
```

Comment:
it is not imported anywhere, is it?

Reply:
No, but I would prefer to keep it there in case we need to generate automatically documentation on which actions are instrumented for each component, and its structure.